src/stores/enums/side.ts

```diff
@@ -0,0 +1,8 @@
+enum Side {
```

Comment:
honestly, I'm not a big fan of separating classes by types :) could you plz move `Side` inside `strores` dir just as a class

plus for me `Up, Down` should be a separate enum called `Dir`